cfme/services/dashboard/ssui.py

```diff
@@ -0,0 +1,163 @@
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ["", "Dashboard"] and
+            self.dashboard_card.is_displayed)
+
+    @property
+    def is_displayed(self):
+        return self.in_dashboard
+
+
+class MyServiceForm(SSUIBaseLoggedInPage):
+
+    service = SSUIlist(list_name='serviceList')
+
+
+class MyServicesView(MyServiceForm):
+    title = Text(locator='//li[@class="active"]')
+
+    def in_myservices(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ["", "My Services"])
+
+    @property
+    def is_displayed(self):
+        return self.in_myservices and self.title.text == "My Services"
+
+
+@Dashboard.total_service.external_implementation_for(ViaSSUI)
+def total_service(self):
+    """Returns the count of total services(Integer) displayed on dashboard"""
+
+    view = navigate_to(self, 'DashboardAll')
+    total_services = view.dashboard_card.get_count('Total Services')
+    view = navigate_to(self, 'TotalServices')
+    view.flash.assert_no_error()
+    view = self.create_view(MyServicesView)
+    assert view.is_displayed
+    return total_services
+
+
+@Dashboard.total_request.external_implementation_for(ViaSSUI)
+def total_request(self):
+    """Total Request cannot be clicked so this method just
+    returns the total number of requests displayed on dashboard.
+    """
+
+    view = navigate_to(self, 'DashboardAll')
+    total_requests = view.dashboard_card.get_count('Total Requests')
+    return total_requests
+
+
+@Dashboard.retiring_soon.external_implementation_for(ViaSSUI)
+def retiring_soon(self):
+    """Returns the count of retiring soon services displayed on dashboard"""
+
+    view = navigate_to(self, 'DashboardAll')
+    retiring_services = view.aggregate_card.get_count('Retiring Soon')
+    view = navigate_to(self, 'RetiringSoon')
+    view.flash.assert_no_error()
+    view = self.create_view(MyServicesView)
+    assert view.is_displayed
+    return retiring_services
+
+
+@Dashboard.current_services.external_implementation_for(ViaSSUI)
+def current_service(self):
+    """Returns the count of active services displayed on dashboard"""
+
+    view = navigate_to(self, 'DashboardAll')
+    current_services = view.aggregate_card.get_count('Current Services')
+    view = navigate_to(self, 'CurrentServices')
+    view.flash.assert_no_error()
+    view = self.create_view(MyServicesView)
+    assert view.is_displayed
+    return current_services
+
+
+@Dashboard.retired_services.external_implementation_for(ViaSSUI)
+def retired_service(self):
+    """Returns the count of retired services displayed on dashboard"""
+
+    view = navigate_to(self, 'DashboardAll')
+    retired_services = view.aggregate_card.get_count('Retired Services')
+    view = navigate_to(self, 'RetiredServices')
+    view.flash.assert_no_error()
+    view = self.create_view(MyServicesView)
+    assert view.is_displayed
```

Comment:
Yes the retired service view from dashbord is same as Myservice view with just the retired services displayed .
I was planning to add those checks later as the PR is required by other cards to start working on .